leash.go

```diff
@@ -113,9 +113,9 @@ func run(options GlobalOptions) {
 			close(realToBeSent)
 		}()
 
-		// start up the sender
-		go sendToLibhoney(realToBeSent, toBeResent, delaySending, doneSending,
-			options.TailSample)
+		// start up the sender. all sources are either sampled when tailing or in-
+		// parser, so always tell libhoney events are pre-sampled
+		go sendToLibhoney(realToBeSent, toBeResent, delaySending, doneSending, true)
```

Comment:
I was on the fence but I think "add it back if we need it again" will make future us happier than "keep around this thing that's sorta confusing and not actually used anywhere".